src/Api/Models/Request/Accounts/UpdateProfileRequestModel.cs

```diff
@@ -8,6 +9,7 @@ public class UpdateProfileRequestModel
         [StringLength(50)]
         public string Name { get; set; }
         [StringLength(50)]
+        [Obsolete("Changes will be made via the 'password' endpoint going forward.")]
         public string MasterPasswordHint { get; set; }
```

Comment:
Do we need to remove this property from any methods or calls? I'm wondering if the MasterPasswordHint will be nulled if something calls the old endpoint and a user's profile is updated

Reply:
I was advised to leave it as is for now for backwards combability. 

That does also mean that the Profile UI's handling of the Password Hint field did not change

We could probably remove it from the UI and accept nulls as a "do not change" flag.

Reply:
Yeah I feel like having it in both places in the UI is confusing. Also, the title of the work item (Move MP hint) suggests that we should remove it from the Profile UI.